src/components/Scheme/SchemeOpenBountyPage.tsx

```diff
@@ -41,9 +41,8 @@ export default class SchemeOpenBounty extends React.Component<IProps, IState> {
   private getApi = async (): Promise<void> => {
     // query all open bounties issued by current dao ordered by decending price
     // rinkeby api and mainnet api are different subdomains
-
     const network = (await getNetworkName()).toLowerCase();
-    const testnet = network === "rinkeby" ? "rinkeby." : "";
+    const testnet = network === "main" ? "" : `${network}.`;
```

Comment:
will this work for xDai?

Reply:
you are asking if we might have the bounty scheme in xdai ?
so ,might be though not now ..and if yes so any how the scheme json will need to be updated.
so I suggest to remove this update.

Reply:
is bounty scheme in kovan?

Reply:
not yet. and if there will be it will require updating the json for this scheme. 
so again , please remove this update

Reply:
leaving this update will require us to verify that the rinkeby bounty scheme is working as before with this change . did you ?

Reply:
The logic is so simple, and the way to test it not trivial (in fact not possible since we can't use rinkeby), that I haven't worried about it.

Reply:
Maybe @EzraWeller can test viewing bounties when he gets to this.